src/main/java/org/humancellatlas/ingest/project/web/ProjectController.java

```diff
@@ -15,10 +15,7 @@
 import org.humancellatlas.ingest.file.FileRepository;
 import org.humancellatlas.ingest.process.Process;
 import org.humancellatlas.ingest.process.ProcessRepository;
-import org.humancellatlas.ingest.project.Project;
-import org.humancellatlas.ingest.project.ProjectEventHandler;
-import org.humancellatlas.ingest.project.ProjectRepository;
-import org.humancellatlas.ingest.project.ProjectService;
+import org.humancellatlas.ingest.project.*;
```

Comment:
It is not a good practice to use start-imports.
Please state explicitly the classes you use.